core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -250,7 +250,8 @@ public void onCreate() {
         taskManager = new PlaybackServiceTaskManager(this, taskManagerCallback);
         mediaPlayer = new PlaybackServiceMediaPlayer(this, mediaPlayerCallback);
 
-        ComponentName eventReceiver = new ComponentName(getPackageName(), MediaButtonReceiver.class.getName());
+        ComponentName eventReceiver = new ComponentName(MediaButtonReceiver.class.getPackage().getName(),
+                MediaButtonReceiver.class.getName());
```

Comment:
The ComponentName parameter in this case seems to be effective only for API < 18.
My computer is somewhat limited so I'm not able to run the emulator, but I believe this should properly handle such cases, as now the receiver is not in the same package as the service.